.travis.yml

```diff
@@ -18,3 +21,7 @@ matrix:
       compiler: clang
     - env: PLATFORM=Win64
       compiler: clang
+    - env: PLATFORM=Win32
+      os: osx
+    - env: PLATFORM=Win64
+      os: osx
```

Comment:
last time I've checked travis said that they have only a few osx builders and therefore one should only build the neccessary. Is that now different?